src/lib/feature-finder.js

```diff
@@ -59,8 +62,8 @@ function parseFeature(featurePath) {
     let file = fs.readFileSync(featurePath, {encoding: 'utf8'});
     return gherkinParser.parse(file).feature;
   } catch (e) {
-    console.log(featurePath + ' could not be parsed from Gherkin, ignoring as a feature file.', e);
-    return {children: []};
+    console.error(featurePath + ' could not be parsed from Gherkin, ignoring as a feature file.', e.message);
```

Comment:
It's a huge callstack that only shows gherkin and bluebird traces. I changed it because it was making the error and the whole output less readable.